src/common_tasks/corpus_candidate_set.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+import json
+from typing import Dict, Sequence
+
+import boto3
+from prefect import task, Parameter
+from sagemaker import Session
+from sagemaker.feature_store.feature_group import FeatureGroup
+from sagemaker.feature_store.inputs import FeatureValue
+
+from utils import config
+
+
+@task()
+def create_corpus_candidate_set_record(
+        id: str,
+        corpus_items: Dict,
+        unloaded_at: datetime.datetime = datetime.datetime.now()
+) -> Sequence[FeatureValue]:
+    return [
+        FeatureValue('id', id),
+        FeatureValue('unloaded_at', unloaded_at.strftime("%Y-%m-%dT%H:%M:%SZ")),
+        FeatureValue('corpus_items', json.dumps(corpus_items)),
```

Comment:
Do we need to validate any properties in the corpus item dict to ensure that it matches the expectations of this record downstream? Additionally should we check to make sure that our size does not exceed 350KB?

https://docs.aws.amazon.com/sagemaker/latest/dg/feature-store-quotas.html